tools/orca/src/stats.rs

```diff
@@ -22,12 +25,44 @@ pub trait DataCollector {
     ) -> Result<(), Error>;
 }
 
-pub struct BasicStatistics {}
+enum OpKind {
+    WriteOp,
+    ReadOp,
+    Other, //TODO! does this make sense?
```

Comment:
Could be OpKind "Auth" since that's what covers those? 
Similor for opkind error.

Reply:
Yeah actually makes sense, I used `Other` because I thought it could cover some other future use cases, but maybe it would be too generic